.github/ISSUE_TEMPLATE/BUG_FRONTEND.md

```diff
@@ -0,0 +1,29 @@
+---
+name: ⚠️ Report Frontend Bug
+about: Report a Frontend bug on Kubeflow Pipelines
+title: "[frontend]: [Bug Name]"
+labels: kind/bug, area/frontend
+---
+
+### Environment
+
+*  How did you deploy Kubeflow Pipelines (KFP)?
+<!-- If you are not sure, here's [an introduction of all options](https://www.kubeflow.org/docs/pipelines/installation/overview/). -->
+*  KFP version: <!-- If you are not sure, build commit shows on bottom of KFP UI left sidenav. -->
+*  KFP SDK version: <!-- Please attach the output of this shell command: $pip list | grep kfp -->
+
+
+### Steps to reproduce
+
+How can we make the problem occur?
+This could be a description, code snippet, video, screenshot, etc.
+
+
+### Expected result
+
+How should the correct behavior be?
```

Comment:
Nit: *What* should the correct behavior be?

Reply:
That is a good point! Applied to this and all other files